portal-kernel/test/unit/com/liferay/portal/kernel/util/LocaleUtilTest.java

```diff
@@ -119,4 +119,19 @@ public void testFromLanguageIdConsistency() {
 		Assert.assertNull(LocaleUtil.fromLanguageId("de_DE", true, false));
 	}
 
+	@Test
+	public void testFromLanguageIdLocaleIsCreatedAndRetrievableWhenNoValidationDone() {
```

Comment:
@marianoalvarosaiz The test name is about "No Validation", but the parameter passed into LanguageUtil is true. Also, please use Assert.assertSame here instead of ==